azavea/raster-foundry
# Feature/lol/verify images

As part of the worker job pipeline, we need to ensure that each tiff has three
    bands. We use gdal to test for this. The first Mb of data has enough data to
    verify the image bands so we get the first million bytes from the tiff on S3
    and store it into a temporary file. This temp file is loaded with gdal and the
    bands are counted. We add a new INVALID status to the layer image enum and
    update the django model if the image cannot be loaded or three or more bands
    are not detected.

Also adds provisioning step to install python dependencies via pip.

Connects #119

### To Test
 * Upload a new image via the UI.
 * Check that a message for the event was added to your SQS queue.
 * Run the worker processor. (see https://github.com/azavea/raster-foundry/pull/136)
 * Check that the image is marked as verified in the database.
